rules/windows/lateral_movement_dns_server_overflow.toml

```
@@ -35,7 +35,7 @@ references = [
 risk_score = 47
 rule_id = "11013227-0301-4a8c-b150-4db924484475"
 severity = "medium"
-tags = ["Elastic", "Network", "Windows"]
+tags = ["Elastic", "Network", "Windows", "Threat Detection", "Lateral Movement"]
```

```suggestion
tags = ["Elastic", "Network", "Threat Detection", "Lateral Movement"]
```
Though this rule is targeting a Windows exploit, it works entirely off of Network data only.